spec/features/find/v2/results/search_results_enabled_spec.rb

```diff
@@ -126,6 +126,37 @@
     end
   end
 
+  context 'when filter by secondary subjects' do
```

Comment:
@Nitemaeric I added to the features folder for now. I will move everything to system spec in another PR not this one. And divide the filters also in the future PR.